srtcore/core.cpp

```diff
@@ -2234,8 +2234,9 @@ bool CUDT::createSrtHandshake(
     return true;
 }
 
-static int FindExtensionBlock(uint32_t *begin, size_t total_length,
-        size_t& w_out_len, uint32_t*& w_next_block)
+template <class Integer>
```

Comment:
This has been changed into `template Integer` because this function is used in the code now once with `uint32_t` and once with `int32_t`. For the extractor it actually doesn't matter.